src/routes/guides/signatures/+page.svelte

```diff
@@ -0,0 +1,157 @@
+    points to the <a href="/tags/metadata/#jwks.json">jwks.json</a>. Once you load that file, you
+    can use the other properties to determine the correct key to verify the signature with.
+  </p>
+
+  <p>
+    Check out the <em>verify_code</em> function in <A
+      href="https://github.com/ioxiocom/ioxio-tags/blob/main/api/app/tag.py">our demo application</A
+    > for a full example.
+  </p>
+
+  <SectionTitle title="Payload" />
+  <p>
+    Once the COSE data is parsed, you will get the properties of the code, so the <em>iss</em>,
+    <em>product</em>, and <em>id</em>, from it.
+  </p>
+
+  <p>
+    The data is signed with <A href="https://cose-wg.github.io/cose-spec/">COSE</A> (CBOR Object Signing
+    and Encryption), effectively a version of the JOSE system but for CBOR, and <A
+      href="https://cbor.io">CBOR</A
+    > itself is a more efficiently packed binary friendly format similar to JSON.
+  </p>
+
+  <SectionTitle title="Parsing process" />
+
+  <ol>
+    <li>Scan the QR code</li>
+    <li>Verify the header - that the first 4 characters match exactly <em>IT1:</em></li>
+    <li>Remove the header from the code</li>
+    <li>Base45 decode the code</li>
+    <li>
+      Parse the <em>alg</em> and <em>kid</em> from the COSE header, as well as the COSE payload
+      <strong>without verifying the signature</strong>
+    </li>
+    <li>
+      Fetch <em>https://{"{"}payload.iss{"}"}/.well-known/product-passport.json</em>
+      <a href="/tags/metadata/#product-passport.json">metadata</a>
+    </li>
+    <li>
+      Fetch the <a href="/tags/metadata/#jwks.json">JWKS keys</a> from <em>{"{"}jwks_url{"}"}</em> in
+      the product-passport.json
+    </li>
+    <li>Verify the COSE signature matches the key provided via JWKS</li>
+    <li>
+      Fetch <em
+        >https://{"{"}payload.iss{"}"}/.well-known/product-passport/{"{"}payload.product{"}"}.json</em
+      >
+      for <a href="/tags/metadata/#%7Bproduct%7D.json">product metadata</a>
+    </li>
+    <li>
+      Fetch <em
+        >https://{"{"}product_dataspace{"}"}.well-known/dataspace/dataspace-configuration.json</em
+      >
+    </li>
+    <li>
+      Proceed to do what you wish with the <em>supported_dataproducts</em> from the
+      <a href="/tags/metadata/#product-passport.json">{"{"}payload.product{"}"}.json</a>, including
+      e.g. fetching the data products via the <em>product_gateway_url</em> from the
+      <em>dataspace-configuration.json</em>
+    </li>
+  </ol>
+
+  <p>
+    Check out our Sandbox's <A
+      href="https://sandbox.ioxio-dataspace.com/.well-known/dataspace/dataspace-configuration.json"
+      >dataspace-configuration.json</A
+    >
+    to see what configuration is available for IOXIO dataspaces.
+  </p>
+
+  <SectionTitle title="Example signed code" />
+  <p>
+    The line feeds are added to make it easier to see what the code is like, but are not a part of
+    the code. Having extra linefeeds breaks the code.
+  </p>
+
+  <Code lang={json}>
```

Comment:
This is not JSON data, language should likely be set to something else; is there some plain text?

Reply:
yep, there's exactly plaintext)